### PR TITLE
Realms mappings

### PR DESCRIPTION
**Pull request by Sollace**
Fills in the mappings that were present in #1155 but missing from #1167

### CODE REVIEW

File: mappings/com/mojang/realmsclient/dto/RealmsServer.mapping [hunk 1]
```diff
@@ -37,9 +37,9 @@ CLASS net/minecraft/class_4877 com/mojang/realmsclient/dto/RealmsServer
 		ARG 1 slots
 	METHOD method_25062 getName ()Ljava/lang/String;
 	METHOD method_25063 parseSlots (Lcom/google/gson/JsonArray;)Ljava/util/Map;
-		ARG 0 jsonArray
+		ARG 0 json
 	METHOD method_25064 setDescription (Ljava/lang/String;)V
-		ARG 1 motd
+		ARG 1 description
```
Comment (liach) [inline mappings/com/mojang/realmsclient/dto/RealmsServer.mapping:42]: `motd` is like the parameter in 20w09a, but replacing it with better ones should be good

File: mappings/com/mojang/realmsclient/dto/ValueObject.mapping [hunk 2]
```diff
@@ -1,3 +1,5 @@
 CLASS net/minecraft/class_4352 com/mojang/realmsclient/dto/ValueObject
-	METHOD method_25094 (Ljava/lang/reflect/Field;)Z
+	METHOD method_25093 getname (Ljava/lang/reflect/Field;)Ljava/lang/String;
```
Comment (liach) [inline mappings/com/mojang/realmsclient/dto/ValueObject.mapping]:
```suggestion
	METHOD method_25093 getName (Ljava/lang/reflect/Field;)Ljava/lang/String;
```

File: mappings/net/minecraft/realms/SizeUnit.mapping [hunk 3]
```diff
@@ -1,4 +1,8 @@
 CLASS net/minecraft/class_4428 net/minecraft/realms/SizeUnit
+	FIELD field_20200 B Lnet/minecraft/class_4428;
+	FIELD field_20201 KB Lnet/minecraft/class_4428;
+	FIELD field_20202 MB Lnet/minecraft/class_4428;
+	FIELD field_20203 GB Lnet/minecraft/class_4428;
```
Comment (liach) [inline mappings/net/minecraft/realms/SizeUnit.mapping]: Aren't these already auto-inferred by the enum automapper?

**Review by liach — APPROVED** [review]
Good fix on bad mojmap names!

File: mappings/net/minecraft/realms/SizeUnit.mapping (hunk 3 above)
Comment (Sollace) [reply mappings/net/minecraft/realms/SizeUnit.mapping]: Apparently not? I wouldn't have mapped them if they had names in Enigma.

Comment (Sollace) [reply mappings/net/minecraft/realms/SizeUnit.mapping]: I looked now and they do indeed have implicit names. They didn't before. The code in Enigma from before looked like:

```
public static final class_4428 field_20200 = new class_4428("B", 0);
```

**Review by Runemoro — APPROVED** [review]

**Comment by Sollace** [comment]
@liach Branch updated

**Comment by liach** [comment]
Actually this isn't really a refactor... doesn't change frequently used stuff at all.

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**